### DISCUSSION TITLE
⚠️ Nueva política de ramas para la Base de conocimiento ⚠️

### DISCUSSION THREAD

**Discussion by CarmenRuizPorcel** [Aparking/BaseConocimiento#1] [Announcements]
## 👋 ¡Hola equipo!

¡Es hora de cambiar las cosas por aquí! 😄 Los profesores nos han dejado claro lo importante que es tener una documentación como código, ¡y nosotros estamos listos para darle caña a eso!

A partir de ahora, vamos a seguir una nueva política de ramas en nuestro repositorio de documentación. Aquí van los detalles:

- ¿Actualizar el reporte de IA? ¡Genial! Solo asegúrate de crear una rama llamada **AI-Report**, actualiza desde allí y luego haz el merge.
- ¿Feedback? ¡Por supuesto! Crea una rama **Feedback/weekX**, donde **'X'** es el número de la semana correspondiente al feedback.
- ¿Nuevos documentos o actualizaciones? ¡Estamos encantados de verlos! Crea una rama **Documents/nombre-del-documento/Y** (Y es el número de la issue).
- Y si necesitas corregir algún error, sigue la política de fixes establecida para el backend y el frontend.

¡Ah, y aquí viene la parte importante! Antes de hacer el merge, asegúrate de que al menos una persona del equipo haya revisado y aceptado la pull request.

¡Pero espera, hay más! Al enviar la pull request, vamos a asegurarnos en las actions de que Docusaurus se despliegue sin problemas en Vercel. ¡No queremos repetir los problemas que tuvimos anteriormente con el despliegue!

¡Vamos equipo, a darle duro y hacer que nuestra documentación brille como nunca! 💪📚

**Comment by perealberto** [comment]
¡Me parece genial! A partir podemos subir el feedback directamente y acceder a él desde el mismo docusaurus, centralizando toda la información en el mismo lugar. 💯🏅

**Reply by laurolmer** [reply]
Perfecto Carmen, muy buena alternativa 🦅

**Comment by virolialo** [comment]
Me parece una buenísima idea tanto para ver quién edita y qué documento, además de saber quién es el revisor o revisores del documento tras subir la PR a la base del conocimiento.

Está genial pensado Carmen, eres una maquina 👍

**Reply by virolialo** [reply]
Esta tarde actualizaré la guía del Docusaurus para que quede por ahí reflejado también cómo hacer uso de la base del conocimiento, así quien consulte ese documento, no tendrá despistes :)

**Comment by CarmenRuizPorcel** [comment]
Se me ha olvidado comentar que para subir las píldoras teóricas, creéis una rama **Theory-Pills/weekX/nombre-pídora** 😺
